specification/sql/resource-manager/Microsoft.Sql/2014-04-01/deprecated.json

```diff
@@ -1,184 +1,187 @@
-              },
-              {
-                  "$ref": "#/parameters/SubscriptionIdParameter"
-              },
-              {
-                  "$ref": "#/parameters/ResourceGroupParameter"
+  "paths":{
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/transparentDataEncryption":{
+          "get":{
+              "tags":[
+                  "TransparentDataEncryption"
+              ],
+              "operationId":"TransparentDataEncryptionConfigurations_ListByDatabase",
+              "description":"Gets a list of a database's transparent data encryption configurations. There is only ever one element, named 'current', so GetTransparentDataEncryptionConfiguration should be used instead.",
+              "deprecated":true,
+              "x-ms-examples":{
+                  "Get a list of transparent data encryption configurations":{
+                      "$ref":"./examples/DatabaseTransparentDataEncryptionList.json"
+                  }
               },
-              {
-                  "$ref": "#/parameters/ServerNameParameter"
+              "parameters":[
+                  {
+                      "$ref":"#/parameters/ApiVersionParameter"
+                  },
+                  {
+                      "$ref":"#/parameters/SubscriptionIdParameter"
+                  },
+                  {
+                      "$ref":"#/parameters/ResourceGroupParameter"
+                  },
+                  {
+                      "$ref":"#/parameters/ServerNameParameter"
+                  },
+                  {
+                      "name":"databaseName",
+                      "in":"path",
+                      "required":true,
+                      "type":"string",
+                      "description":"The name of the database for which the transparent data encryption applies."
+                  }
+              ],
+              "responses":{
+                  "200":{
+                      "description":"OK",
+                      "schema":{
+                          "$ref":"#/definitions/TransparentDataEncryptionListResult"
+                      }
+                  }
               },
-              {
-                  "name": "databaseName",
-                  "in": "path",
-                  "required": true,
-                  "type": "string",
-                  "description": "The name of the database to import into"
+              "x-ms-pageable":{
+                  "nextLinkName":null
               }
-          ],
-          "responses": {
-              "200": {
-                  "description": "OK",
-                  "schema": {
-                  "$ref": "#/definitions/ExtensionListResult"
+          }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/extensions":{
+          "put":{
```

Comment:
why we use PUT to retrieve the extensions? should be GET right?

Reply:
Oops, yes that's a mistake. Thanks for spotting.